scan_offset parameter for Ros2 (#80)
ros-perception/depthimage_to_laserscan: README.md

* `scan_time` (float) - The time in seconds between scans to report to the consumer of the LaserScan message.  This is set directly in the published message.  Defaults to 0.033 seconds.
* `range_min` (float) - The minimum distance in meters a projected point should be.  Points closer than this are discarded.  Defaults to 0.45 meters.
* `range_max` (float) - The maximum distance in meters a projected point should be.  Points further than this are discarded.  Defaults to 10.0 meters.
* `scan_height` (int) - The number of rows from the depth image to use for the laser projection.  Defaults to 1.

> * `scan_offset` (float) - The height ratio of the depth image defining on what height the depth data should be taken from (0.0=top row of image, 1.0=bottom row). Defaults to 0.5. 

Reviewer: ```suggestion
* `scan_offset` (float) - The height ratio of the depth image defining which height the depth data should be taken from (0.0=top row of image, 1.0=bottom row). Defaults to 0.5. 
```
